Resources/Prototypes/Theta/Shipevent/Structures/shields.yml

```
@@ -43,7 +43,7 @@
   suffix: Theta
   components:
     - type: CircularShield
-      consumptionPerM2: 1
+      consumptionPerM2: 0.35 # too much at 1
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)